tests/test_tui.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Henrysk
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import pathlib
+import sys
+
+import pytest
+sys.path.append(str(pathlib.Path(__file__).parent.parent / "src" /"app")) #add the directory of the app
```

Comment:
Nel mio sistema (Linux) 'app' non va bene, perchè la directory è 'App'

```suggestion
sys.path.append(str(pathlib.Path(__file__).parent.parent / "src" /"App")) #add the directory of the app
```

Reply:
Forse sei partito da un commit vecchio e la cartella `app` si chiama ancora `App`.
Se necessario provo con WSL e a cercare di risolvere la problematica se esiste.

Reply:
GitHub dice ancora che si chiama 'App'. Se sei da Windows, sicuramente git non nota la differenza e le considera uguali. Se lo fai con WSL penso che dovresti riuscire a cambiare nome.

Reply:
Ho appena provato, in effetti si chiama ancora `App`.
Ora lo sistemo